experimental/segmenter/src/line.rs

```diff
@@ -159,6 +159,31 @@ pub type LineBreakIteratorUtf16<'l, 's> = LineBreakIterator<'l, 's, LineBreakTyp
 /// assert_eq!(&breakpoints, &[6, 11]);
 /// ```
 ///
+/// The segmenter returns mandatory breaks (as defined by [definition LD7][LD7] of
+/// Unicode Standard Annex #14, _Unicode Line Breaking Algorithm_) as well as
+/// line break opportunities ([definition LD3][LD3]).
+/// It does not distinguish them.
```

Comment:
suggestion: _may_ be worth saying that it is possible for callers to distinguish post-facto by checking the characters against a very small list, since learning that requires some poking around the spec.

Reply:
Done, with an example.